deb/build/debian/jenkins.postrm

```diff
@@ -4,8 +4,16 @@ set -e
 
 case "$1" in
 purge)
-	userdel @@ARTIFACTNAME@@ || true
-	groupdel @@ARTIFACTNAME@@ || true
+	if [ -x "$(command -v deluser)" ]; then
+		deluser --quiet --system @@ARTIFACTNAME@@ >/dev/null || true
+	else
+		userdel @@ARTIFACTNAME@@ || echo "failed to remove @@ARTIFACTNAME@@ system user" >&2
+	fi
+	if [ -x "$(command -v delgroup)" ]; then
+		delgroup --quiet --system --only-if-empty @@ARTIFACTNAME@@ >/dev/null || true
+	else
+		groupdel @@ARTIFACTNAME@@ || echo "failed to remove @@ARTIFACTNAME@@ system group" >&2
+	fi
```

Comment:
Following the advice from [this page](https://wiki.debian.org/AccountHandlingInMaintainerScripts).